ex12.py

```diff
@@ -0,0 +1,14 @@
+import numpy as np
+
+
+def remove_dimension(matrix):
+    """
+    A NumPy program to remove single-dimensional entries from a specified shape.
+    :param matrix:
+    :return: the matrix without the single-dimensional entries
+    """
+    return np.squeeze(matrix).shape
```

Comment:
You claimed that you are returning a matrix but you are actually returning it's shape (tuple).